signingscript/script.py

```diff
@@ -24,35 +19,23 @@
 statsd.namespace = 'releng.scriptworker.signing'
 
 
-# SigningContext {{{1
-class SigningContext(Context):
-    """Status and configuration object."""
-
-    signing_servers = None
-
-    def __init__(self):
-        """Initialize SigningContext."""
-        super(SigningContext, self).__init__()
-
-    def write_json(self, *args):
```

Comment:
We don't want to `write_json` in the *scripts - this is a hack to save a copy of the running task to disk. If the scripts do this as well, we'll overwrite.

Maybe we need a `ClientContext` in scriptworker?

Reply:
Good idea. I'll follow up in scriptworker.

Reply:
Done at https://github.com/mozilla-releng/scriptworker/pull/196/files#diff-fdca3340e9146ac40239a7ef6df5ba66R162